runners/core-java/build.gradle

```diff
@@ -41,17 +41,17 @@ test {
 dependencies {
   compile project(path: ":model:pipeline", configuration: "shadow")
   compile project(path: ":sdks:java:core", configuration: "shadow")
-  compile project(path: ":model:fn-execution", configuration: "shadow")
+  compile project(path: ":model:job-management:", configuration: "shadow")
   compile project(":runners:core-construction-java")
-  compile project(":sdks:java:fn-execution")
+  permitUsedUndeclared project(":sdks:java:fn-execution")
```

Comment:
We should not need `permitUsedDeclared` for this. It seems like it was OK before. Otherwise at least it would be `runtime`.